src/ui/map.js

```diff
@@ -810,6 +812,7 @@ class Map extends Camera {
 
     /**
      * Returns true if the map is panning, zooming, rotating, or pitching due to a camera animation or user gesture.
+     * @returns {Boolean} True if the map is moving.
```

Comment:
Another example of the general description stealing the `@return` value's thunder.